### ISSUE TITLE
Get particle from atom?

### ISSUE THREAD

**Issue by aizvorski** [openmm/openmm#3680]
I'm wondering if there is any way to relate application-level objects (eg `app.Atom`) to library-level objects (eg particle).

For example:
```
for atom in topology.atoms():
    atom_index = ??? # index of particle for atom
    mass = system.getParticleMass(atom_index)
```

At least when creating a system in the standard way, are the particles created by ForceField `createSystem(topology, ...)` guaranteed to be the same number and in the same order as the atoms in the topology?

**Comment by peastman** [comment]
Correct.  The order is identical.  `atom.index` gives the correct index of the particle.

**Comment by aizvorski** [comment]
@peastman Something to add to the docs?  I assumed this, and seemed to be true in simple tests, but didn't have any way to know if it's guaranteed.

**Comment by peastman** [comment]
Where do you suggest adding it?

**Comment by aizvorski** [comment]
Could be under ForceField createSystem - something like 

"The constructed system contains one particle for each atom in the topology, and the particles are in the same order as the atoms"

It may be good to add a similar note under Topology, "when a system is constructed from a topology using a forcefield, the atoms are converted to corresponding particles in the system where the particles are in the same order and have the same indexes"

**Comment by peastman** [comment]
Sounds good.  I added the `documentation` label.  A PR would be welcome!
